supply/lib/supply/uploader.rb

```diff
@@ -130,6 +132,10 @@ def upload_binaries
         apk_version_codes.push(upload_binary_data(apk_path))
       end
 
+      return apk_version_codes
+    end
+
+    def upload_binaries(apk_version_codes)
```

Comment:
Can we rename this to `upload_mapping`? "mapping" seems more appropriate then "binary" here 🙃

Reply:
I fixed at 49e2b1d 😃